File: COTE/LeetCode/easy/FindNumbersWithEvenNumbers.py
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        cnt=0
        for n in nums:
            if len(str(n))%2==0:
                cnt+=1
        return cnt


# sum(n~ for n in arr)
# print(2%2==0) ...True
# arr=[True,True,False]
# print(sum(arr)) ...2
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        return sum(len(str(n))%2==0 for n in nums)

File: COTE/LeetCode/easy/FindNumbersWithEvenNumbers.js
https://leetcode.com/explore/learn/card/fun-with-arrays/521/introduction/3237/

/**
 * @param {number[]} nums
 * @return {number}
 */
 var findNumbers = function(nums) {
   let lengthList=nums.map(el=>el.toString().length)
   return lengthList.reduce((cnt,e)=>{
       if(parseInt(e)%2===0){
           return cnt+=1
       }
       return cnt
   }, 0)
};

//s1
var findNumbers = function(nums) {

   const isEvenNums = num => {
       return String(num).length % 2 === 0;  
   };
   
   return nums.filter(elem => isEvenNums(elem)).length;
   
};